package/Dockerfile

```diff
@@ -13,7 +30,7 @@ RUN mkdir /root/.kube && \
     ln -s /usr/bin/rancher /usr/bin/ensure-default-admin
 WORKDIR /var/lib/rancher
 
-ARG ARCH=amd64
+ARG ARCH
```

Comment:
Why get rid of the default value?

Reply:
No particular reason, I'm adding it back.